src/Microsoft.PowerApps.TestEngine/PowerApps/PowerAppFunctions.cs

```diff
@@ -67,6 +67,15 @@ private async Task<bool> CheckIfAppIsIdleAsync()
         {
             try
             {
+                var checkJSSDKExistExpression = "typeof PowerAppsTestEngine";
+                var result = await _testInfraFunctions.RunJavascriptAsync<string>(checkJSSDKExistExpression);
+                if (result == "undefined")
```

Comment:
nit: I doubt it will be an issue, but you might want to run a `.toLower()` on `result` just in case, for some reason, `undefined` comes back in a slightly different case.